functions/__init__.py

```diff
@@ -17,59 +28,46 @@ def init_db(user_code):
                   order_index INTEGER DEFAULT 0
                 )''')
     conn.commit()
-    conn.close()
-
 
 def add_image_to_db(user_code, filename):
-    conn = sqlite3.connect('uploads/database.db')
+    conn = Database.get_connection()
     c = conn.cursor()
     c.execute(f'''SELECT 1 FROM "{user_code}" WHERE filename = ?''', (filename,))
     exists = c.fetchone()
     if not exists:
         c.execute(f'''INSERT INTO "{user_code}" (filename) VALUES (?)''', (filename,))
         conn.commit()
-    conn.close()
 
 def remove_image_from_db(user_code, filename):
-    conn = sqlite3.connect('uploads/database.db')
+    conn = Database.get_connection()
     c = conn.cursor()
     c.execute(f'''UPDATE "{user_code}" SET visible = 0 WHERE filename = ?''', (filename,))
     conn.commit()
-    conn.close()
 
 def get_visible_images(user_code):
-    conn = sqlite3.connect('uploads/database.db')
+    conn = Database.get_connection()
     c = conn.cursor()
     c.execute(f'''SELECT filename FROM "{user_code}" WHERE visible = 1 ORDER BY order_index, id''')
     images = [row[0] for row in c.fetchall()]
-    conn.close()
     return images
 
 def update_image_order(user_code, filename, new_index):
-    conn = sqlite3.connect('uploads/database.db')
+    conn = Database.get_connection()
     c = conn.cursor()
-    # Получаем текущий список файлов с их порядковыми индексами
-    c.execute(f'''SELECT filename, order_index FROM "{user_code}" WHERE visible = 1 ORDER BY order_index, id''')
-    files = c.fetchall()
-
-    # Создаем временный список для обновления порядка
-    updated_files = []
-    for f in files:
-        if f[0] == filename:
-            continue
-        updated_files.append(f)
-
-    # Вставляем перемещенный файл в новую позицию
-    updated_files.insert(new_index, (filename, 0))  # 0 - временное значение для order_index
-
-    # Обновляем order_index всех файлов
-    for index, file in enumerate(updated_files):
-        c.execute(f'''UPDATE "{user_code}" SET order_index = ? WHERE filename = ?''', (index, file[0]))
-
+    
+    c.execute(f'''SELECT order_index FROM "{user_code}" WHERE filename = ?''', (filename,))
+    current_index = c.fetchone()[0]
```

Comment:
Fine, but there is a nicer way to write this with tuple unpacking:
```python
current_index, = c.fetchone()
```

One advantage of tuple unpacking over your solution is that if there is more fields selected than needed by the Python code, your solution ignores this, while the tuple unpacking will raise a `TypeError`